data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/special_alignment.cfg

```diff
@@ -0,0 +1,246 @@
+#textdomain wesnoth-test
+
+{GENERIC_UNIT_TEST "special_alignment_test" (
```

Comment:
Please update the tests to use the new macros that have been used in more recent PRs.

Reply:
done